astropy/coordinates/representation.py

```diff
@@ -3006,6 +3010,9 @@ def to_cartesian(self, base):
         return self.d_distance * base.represent_as(
             UnitSphericalRepresentation).to_cartesian()
 
+    def norm(self, base=None):
```

Comment:
Need a docstring? Also, looks like this also needs a change log for 4.3. Thanks!

Reply:
It will inherit the docstring from `BaseDifferential.norm()`.  I followed the same approach as taken for the other overridden methods in `RadialDifferential`.